docs/tutorials/intro_tutorial.ipynb

```diff
@@ -604,44 +606,31 @@
    "source": [
     "#### Moving in Mesa\n",
     "\n",
-    "Under the hood, each agent's position is stored in two ways: the agent is contained in the grid in the cell it is currently in, and the agent has a `pos` variable with an (x, y) coordinate tuple. The `place_agent` method adds the coordinate to the agent automatically.\n",
+    "Due to the complex dynamics of space and movement, Mesa offers a wide range space options and has built a structure to allow for the addition of even more spaces or custom user space creation.\n",
     "\n",
-    "Now we need to add to the agents' behaviors, letting them move around and only give money to other agents in the same cell.\n",
+    "The two main approaches to space are discrete space (think cells that agents occupy of various types) and continuous space (how we exist). For this tutorial we will use discrete space and [Moore neighborhood](https://en.wikipedia.org/wiki/Moore_neighborhood). A Moore neighborhood means agents can interact with 8 neighbors. To find out more about the other options and what they can do, check out the [Discrete Space API](https://mesa.readthedocs.io/latest/apis/discrete_space.html)\n",
     "\n",
-    "Mesa has a  built-in `get_neighborhood` method, which returns all the neighbors of a given cell. This method can get two types of cell neighborhoods: [Moore](https://en.wikipedia.org/wiki/Moore_neighborhood) (includes all 8 surrounding squares), and [Von neumann](https://en.wikipedia.org/wiki/Von_neumann_neighborhood)(only up/down/left/right). It also needs an argument as to whether to include the center cell itself as one of the neighbors.\n",
+    "<br><br>\n",
+    "![Discerete Space Diagram](../images/Discrete_Space.drawio.png)\n",
     "\n",
-    "With that in mind, the agent's `move` method looks like this:\n",
+    "Due to the variety of spaces and their own unique needs, we must now instantiate our **MoneyAgents** as CellAgents. CellAgents is a supplmentary class to mesa agent class that is specifically built to interact and move within discretespace.  \n",
```

Comment:
```suggestion
    "Due to the variety of spaces and their own unique needs, we must now create our **MoneyAgents** as a subclass of CellAgents. CellAgents is a subclass of the default `Agent` class that is specifically built to interact and move within discrete spaces.  \n",
```

Reply:
updated to correctly represent `CellAgent` in the adding space module